packages/core/botpress/src/cli/templates/init-default/src/index.js

```diff
@@ -45,8 +45,13 @@ module.exports = async bp => {
   /// Conversation Management
   ////////////////////////////
 
+  // Welcome screen
+  bp.hear({ platform: 'facebook', type: 'postback', text: 'GET_STARTED' }, (event, next) => {
+    bp.dialogEngine.processMessage(event.sessionId || event.user.id, event).then()
+  })
```

Comment:
@dmk23 , not sure it makes sense here since by default you don't have messenger module installed on fresh init. Maybe it makes more sense to leave this code commented out?

Reply:
@epaminond We could leave it commented out, but another option is to add a step to fresh init to ask about channels and generate these lines conditionally. Though even if this line is present for non-Facebook installs, I am not seeing how it would break anything... BP would just listen for the event that will never come

On the other hand, it might make sense to listed for other postbacks, not just `get_started`. I am about to implement them in other parts of my bot and could share more suggestions/ commits